src/main/webapp/content/scss/partials/_report-management.scss

```diff
@@ -106,6 +107,14 @@
   {
     outline: none !important
   }
+  .configuration{
+    .fbibox-content{
+      background-color: #ffffff;
```

Comment:
this color also needs to be part of _variable.scss file...so any time we can change the app colors...